packages/emotion-primitives/src/emotion-native.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import transform from 'css-to-react-native'
+import cssToObjects from 'css-to-object'
```

Comment:
probably should be `cssToObject`

Reply:
Not using this anymore! Haha It's better if you don't review this now 😅